### ISSUE TITLE
Default error message language when submitting with ajax in a headless setup

### ISSUE THREAD

**Issue by jan-thoma** [verbb/formie#1545]
### Question

We use craftcms headless. our frontend is Vue.js when i send an ajax request in this manner to submit a form:

```
const data = new FormData(event.target)

data.append('CRAFT_CSRF_TOKEN', 'XXX')
data.append('handle', 'contact_form')
data.append('siteId', 1)

const request = await fetch('https://demo.com/actions/formie/submissions/submit', {
    method: 'post',
    body: data,
    headers:
    {
        Accept: 'application/json',
        'X-Requested-With': 'XMLHttpRequest'
    }
})
```

The response error fields are always in the language of my main site no matter what siteId i submit with the form. Did missed something here? Also it looks like the default required message is not editable

**Comment by engram-design** [comment]
Right, this is a good point in that Formie will be using the [static translation](https://craftcms.com/docs/4.x/sites.html#static-message-translations) based on the value in `lang/formie.php`. For a headless request however, that's not really applicable (although I've not verified that). But we certainly don't use the `siteId` param to determine how to translate strings - that's all done by Craft itself, but would be based on the request coming as a site request.

I'll need to take a look at some options.

**Comment by engram-design** [comment]
Updated in [4.0.0-beta.8](https://github.com/verbb/formie/releases/tag/4.0.0-beta.8).
